src/string.cr

```diff
@@ -3471,10 +3471,8 @@ class String
   # (characters beyond this point are ignored).
   #
   # ```
-  # "Hello, World".rindex('o')    # => 8
-  # "Hello, World".rindex('Z')    # => nil
-  # "Hello, World".rindex("o", 5) # => 4
-  # "Hello, World".rindex("W", 2) # => nil
+  # "Hello, World".rindex('o') # => 8
+  # "Hello, World".rindex('Z') # => nil
```

Comment:
**question:** Why remove the examples demonstrating the usage of the offset parameter?

Reply:
I didn't intend to remove the offset examples, I removed them because they contained a string in the `char` method.

I'll add them back in :D